### DISCUSSION TITLE
How to disable ssl verification

### DISCUSSION THREAD

**Discussion by jesserota** [thunderclient/thunder-client-support#936] [Q&A]
How to disable ssl verification?

**Comment by rangav** [comment]
@jesserota ssl verification is not mandatory, it’s already disabled 

are you facing any errors ? Can you share screenshot

**Comment by bombero2020** [comment]
![image](https://github.com/rangav/thunder-client-support/assets/40033461/53e921f9-48d2-46af-a132-c613001f97bb)

guest i got the same issue..

**Reply by rangav** [reply]
Hi @bombero2020 are you custom ssl certificates?

**Reply by bombero2020** [reply]
yes,

**Reply by rangav** [reply]
Could be path issue

see example
https://github.com/rangav/thunder-client-support#manual-ssl-certificates

try different paths and see which one works

**Reply by mesketh** [reply]
Hi there - I noticed that latest version is using Axios dep for http work. This supports (via the HttpsAgent) an attribute to `rejectUnauthorized`. Can we expose that in order to disable? This is useful for lower envs where TLS just gets in the way.

**Reply by rangav** [reply]
@mesketh the axios library already configured to `rejectUnauthorized: false`

do you still have issues?

**Reply by qwatts-dev** [reply]
I was able to bypass this same issue by changing the setting for **Thunder-client: Http Library** from `axios` to `got`

![image](https://github.com/rangav/thunder-client-support/assets/54119641/ccb75244-d819-4a44-af98-ec233d1d893a)

It adds this additional property to the `settings.json`:
![image](https://github.com/rangav/thunder-client-support/assets/54119641/229c5bbd-fa8d-4fbf-ae05-b86598845cb2)

**Comment by mesketh** [comment]
> @mesketh the axios library already configured to `rejectUnauthorized: false`
> 
> do you still have issues?

hmm.. interesting - didn't seem to be working for me. Anyway - I'm only a casual user for some quick testing but, thanks for the reply.
